frontend/src/components/material/MaterialLists.vue

```diff
@@ -49,9 +72,23 @@ export default {
     materialLists() {
       return this.camp.materialLists()
     },
+    materialItems() {
+      return this.camp.periods().items.flatMap((period) => period.materialItems().items)
+    },
     materailListsSorted() {
       return materialListsSorted(this.materialLists.allItems)
     },
+    unassignedCount() {
```

Comment:
just a question: would it be possible to count the material items with material list null?

Reply:
As discussed, this is fine for now.